src/renderable/GUI.js

```diff
@@ -146,16 +160,6 @@
         },
 
         /**
-         * function callback for the pointerup event
-         * @ignore
-         */
-        release : function (event) {
-            this.released = true;
-            me.timer.clearTimeout(this.holdTimeout);
-            return this.onRelease(event);
-        },
-
-        /**
```

Comment:
Right question!
When i was working on the GUI_Object, i also added moved, entered and leaved function, and i moved all of them together with clicked and release to the top.
On the bottom i brought together the onClick, onRelease and the (now missing) onMove.
After some fiddling i removed unnecessary functions, but the order was left changed.